examples/MQ/serialization/ExHeader.h

```diff
@@ -0,0 +1,35 @@
+/********************************************************************************
+ * Copyright (C) 2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH       *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *              GNU Lesser General Public Licence (LGPL) version 3,             *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
+
+#ifndef EXHEADER_H
+#define EXHEADER_H
+
+#include "FairMQ.h"   // for fair::mq::Message
+
+namespace boost::serialization {
+class access;
+}
+
+struct ExHeader
+{
+    int eventNumber = 0;
+    int detectorId = 0;
```

Comment:
I don't care much, but didn't we have some "member variables start with `f`" thing?

Reply:
I sometimes make exceptions for the tiny structs. Can't explain why...
I'll follow the rule if there is one :)

Reply:
> but didn't we have some "member variables start with f" thing?

There is no required hungarian notation AFAIK. FairRoot loosely adopted naming rules from ROOT, but there are already violating cases in various places.

That said, I personally dislike this hungarian notation and also don't see a technical reason to have it, so if we would put it up for a vote, I'd vote against having it.

> Can't explain why...

If we voted for having the f-prefix required, I would want to propose an exception for public member variable names which I think is roughly the case when Alexey would intuitively decide to use plain variable names.

Reply:
> public member variable names

Sounds like a good condition for an exception to me :)

Reply:
:+1: for the public member variable exception.

I'll resolve this with exactly that exception!